sphinxcontrib/chapeldomain/chapel.py

```diff
@@ -27,6 +27,37 @@ class ChapelLexer(RegexLexer):
     aliases = ['chapel', 'chpl']
     # mimetypes = ['text/x-chapel']
 
+    known_types = ('bool', 'bytes', 'complex', 'locale', 'imag', 'int',
```

Comment:
Minor nit, but `locale` should be alphabetized after `int` :)

Reply:
How'd it get there?! Fixed